tests/UI/pages/commonPage.js

```diff
@@ -174,7 +174,9 @@ module.exports = class CommonPage {
     await page.click(selector, {clickCount: 3});
     // Delete text from input before typing
     await page.press(selector, 'Delete');
-    await page.type(selector, value);
+    if (value !== ' ') {
+      await page.type(selector, value);
+    }
```

Comment:
I don't like this. If one day we want to write a space in the input, how could we do it ? This logic would skip it.
It would be better to check if it's null I think ?
```suggestion
    if (value !== null) {
      await page.type(selector, value);
    }
```

Reply:
Done :+1: